LayoutTests/imported/blink/fast/repaint/border-radius-repaint-2.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```

Comment:
New tests should not go into an `imported` directory (unless that's `web-platform-tests` and you plan to export back to WPT).

Reply:
Yes, this test is currently not on WPT so @brentfulgham suggested we could import it and then maybe export it to WPT as you mentioned. Should we create a new test?

Reply:
I've decided to create a new similar test, that is a bit more reduced (we don't need the 2 extra divs) and that uses our repaint.js file for the repainting instead.